src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/FixAllContextHelper.cs

```diff
@@ -50,6 +54,48 @@ internal static class FixAllContextHelper
 
                     break;
 
+                case FixAllScope.ContainingMember or FixAllScope.ContainingType:
+                    if (document != null && !await document.IsGeneratedCodeAsync(cancellationToken).ConfigureAwait(false))
+                    {
+                        if (!fixAllSpans.IsEmpty)
+                        {
+                            // We are processing a span-based FixAllContext created for a partial declaration.
+                            var documentDiagnostics = ImmutableArray<Diagnostic>.Empty;
```

Comment:
```suggestion
                            using var _ = ArrayBuilde<Diagnostic>.GetInstance(out var documentDiagnostics);
```

Reply:
Same comment as above https://github.com/dotnet/roslyn/pull/59989#discussion_r821317353. We don't need to optimize for multiple fix all spans within a document case here.